DFC.Digital/DFC.Digital.Web.Core/Config/WebCoreAutofacConfig.cs

```diff
@@ -96,7 +96,7 @@ private static void RegisterAutomapper(ContainerBuilder builder, IEnumerable<Ass
                     cfg.AddProfile(profile);
                 }
             })).AsSelf().InstancePerLifetimeScope();
```

Comment:
This needs changing from This should remain as **InstancePerLifetimeScope()** to **SingleInstance()**

Reply:
This is the change that would have had the biggest impact in testing as we are no longer configuring automapper for each instance requested.